packages/client/src/objectSet/ObjectSetListenerWebsocket.ts

```diff
@@ -527,7 +465,8 @@ export class ObjectSetListenerWebsocket {
       const response = responses[i];
       switch (response.type) {
         case "error":
-          sub.listener.onError(response.error);
+          // sub.listener.onError(response.error);
```

Comment:
did you mean to remove this?

Reply:
I need to delete the comment sorry, I added the replacement down below